twiggy/tests/tests.rs

```diff
@@ -90,6 +90,9 @@ macro_rules! test {
     }
 }
 
+// Top Tests:
+// ----------------------------------------------------------------------------
```

Comment:
Want to split these out into submodules instead?

Reply:
That was the plan, I was having some trouble doing that. Should the `test` macro be duplicated across each file, or is there a way to import that into each submodule after defining it in one place?

Reply:
You have to add a directory nesting level a little deeper than you might normally expect, similar to `wasm-pack/tests/**`:

```
~/wasm-pack
$ tree tests/all/
tests/all/
├── build.rs
├── main.rs
├── manifest.rs
├── readme.rs
└── utils
    ├── file.rs
    ├── fixture.rs
    ├── manifest.rs
    └── mod.rs
```

The key is that every `tests/*.rs` is its own test crate, so you have to nest into a directory that has a `main.rs` that declares all the modules and all that. The test macro should be able to be defined at the top of `main.rs` and then used in every submodule, I think.